data_steward/analytics/cdr_ops/clean_rdr_export_qc.py

```diff
@@ -1062,89 +1090,7 @@
 
 # -
 
-# # COPE survey mapping
```

Comment:
@brendagutman why is this check being removed? can you provide some context on this please.

Reply:
Yes. This check was added when we expected that the cope concepts would be remapped to individual module concepts in Athena. We decided not to go on this route. Instead using custom concepts, the cope_survey_semantic_version_map, and then the survey_conduct table to determine the cope version. 

This check will always fail(cause confusion) and the check  `Survey version and dates` verifies our current cope mapping strategy.

Reply:
Thanks for the context.